src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -66,7 +67,7 @@ const validateSubmitFormFeedbackParams = celebrate({
  */
 export const submitFormFeedback: RequestHandler<
   { formId: string },
-  ErrorDto | PrivateFormErrorDto,
+  FormFeedbackResponseDto | ErrorDto | PrivateFormErrorDto,
```

Comment:
this is no longer correct, right? the success case seems to now be `{ message: string }`

Reply:
thanks! edited